hash_practice/exercises.py

```diff
@@ -2,18 +2,55 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(nk log k)
+        Space Complexity: O(n)
+
     """
-    pass
+    """ not sure why split() doesn't work:
+    # for word in strings:
+    #     split_word ="".join(word.split("").sort()) #aet

```

Comment:
From what I can tell, `split("")` causes a `ValueError` to be thrown for passing in an empty string as a separator. Seems to be a limitation of the function.

You can achieve the effect you want with `list(word)`, which returns a list with the characters of `word` as its members.